hoover/site/settings/common.py

```diff
@@ -107,6 +107,7 @@
 _hour = 60 * _minute
 HOOVER_TWOFACTOR_INVITATION_VALID = 30 # minutes
 HOOVER_TWOFACTOR_AUTOLOGOUT = 3 * _hour
-HOOVER_RATELIMIT_USER = (30, _minute)
+HOOVER_TWOFACTOR_RATELIMIT = None
```

Comment:
This is getting complex really quick.
I think we need some docs in the `hoover.contrib.ratelimit` and/or `hoover.contrib.twofactor` apps regarding this new config.

Either way, with all these new configurations, we're heading towards that `./bin/hoover doctor` you've been talking about.